aif360/sklearn/datasets/__init__.py

```diff
@@ -12,4 +12,7 @@
 from aif360.sklearn.datasets.openml_datasets import fetch_adult, fetch_german, fetch_bank
 from aif360.sklearn.datasets.compas_dataset import fetch_compas
 from aif360.sklearn.datasets.meps_datasets import fetch_meps
-from aif360.sklearn.datasets.tempeh_datasets import fetch_lawschool_gpa
+
+import pkgutil
+if pkgutil.find_loader("tempeh"):
+    from aif360.sklearn.datasets.tempeh_datasets import fetch_lawschool_gpa
```

Comment:
what is the benefit of this method instead of the try, except when actually importing tempeh?

Reply:
Yeah... no benefits over imports in try/except - changed in latest